editoast/src/views/rolling_stocks.rs

```diff
@@ -788,4 +806,59 @@ pub mod tests {
         };
         assert_eq!(response_body, expected_body);
     }
+
+    #[rstest]
+    async fn try_to_delete_rolling_stock_that_is_being_used(
+        #[future] train_schedule_with_scenario: TrainScheduleProjectStudyScenarioTimetableInfraPathRollingStockFixture,
+    ) {
+        let app = create_test_service().await;
+        let train_schedule_with_scenario = train_schedule_with_scenario.await;
+        let rolling_stock_id = train_schedule_with_scenario.rolling_stock.id();
+        let response = call_service(
+            &app,
+            TestRequest::delete()
```

Comment:
You can use `rolling_stock_delete_request(rolling_stock_id)` ;)

Reply:
right :)